profiles/2019-03-01-hybrid/azure/src/main/java/com/microsoft/azure/management/profile_2019_03_01_hybrid/CustomCipherSuites.java

```diff
@@ -15,12 +15,12 @@
 public class CustomCipherSuites {
     // Copy from OkHttp https://github.com/square/okhttp/blob/parent-3.12.8/okhttp/src/main/java/okhttp3/ConnectionSpec.java#L63
     public static final CipherSuite[] APPROVED_CIPHER_SUITES = new CipherSuite[] {
-        // TLSv1.3
-        CipherSuite.TLS_AES_128_GCM_SHA256,
-        CipherSuite.TLS_AES_256_GCM_SHA384,
-        CipherSuite.TLS_CHACHA20_POLY1305_SHA256,
-        CipherSuite.TLS_AES_128_CCM_SHA256,
-        // CipherSuite.TLS_AES_256_CCM_8_SHA256, // Not Found
+        // TLSv1.3 // Not provide as enum for 3.11.0
+        CipherSuite.forJavaName("TLS_AES_128_GCM_SHA256"),
```

Comment:
are there tests that should be added that correspond with these 5 string changes?

Reply:
The is actually what okhttp adds cipher suite. I don't know how to add unit test will special dependency version (okhttp 3.11.0). And I have test with it successfully for whole profile.